io/csv/include/io/csv_io.hpp

```diff
@@ -450,7 +450,7 @@ namespace detray
     auto
         sort_and_remove_duplicates = [](std::map<scalar, std::vector<dindex>> &att) -> dvector<scalar>
     {
-      std::vector<scalar> keys;
+      dvector<scalar> keys;
```

Comment:
We will have to clear this out anyways, *think* while reading we can use std classes, but not when building ...

Reply:
True, I have to decide where I actually need the aligned allocator and where it is Ok to use std::allocator. My guess is this corresponds to the cases where vecmem is needed, too